File: TUAN_2/1.4.2.sum.cpp

#include <iostream>
using namespace std;

int sum_1_to_n(int n) {
    int s = 0;
    for (int i = 1; i <= n; i++) {
        s += i;  // Cộng dồn giá trị của i vào s
    }
    return s;  // Trả về tổng
}

int main() {
    int n;
    cout << "Nhap n: ";  // Hướng dẫn người dùng nhập n
    cin >> n;  // Nhập giá trị n
    int result = sum_1_to_n(n);  // Gọi hàm để tính tổng
    cout << "Tong 1..n = " << result << endl;  // In kết quả
    return 0;  // Kết thúc chương trình
}

File: TUAN_2/1.4.2.sum.jl
function sum_1_to_n(n)
    s = 0
    for i in 1:n
        s += i  # Cộng dồn giá trị của i vào s
    end
    return s  # Trả về tổng
end

println("Nhap n: ")
n = parse(Int, readline())  # Nhập giá trị n từ người dùng
result = sum_1_to_n(n)  # Gọi hàm để tính tổng
println("Tong 1..n = ", result)  # In kết quả